### ISSUE TITLE
Cryptotracker app does not display correct prices

### ISSUE THREAD

**Issue by jsakas** [tidbyt/community#781]
Heads up @ethanfuerst it looks like the app is not displaying correct prices.

Seems like its frozen in time... currently stuck on BTC at $21,214 -0.43% but as of this morning BTC is trading around $17k.

**Comment by ethanfuerst** [comment]
Good catch, will look in to this today

**Comment by ethanfuerst** [comment]
@jsakas Got a chance to dig in to this - Looks like the [API call](https://www.alphavantage.co/documentation/#crypto-intraday) that I used to get coin prices was moved to the premium tier and the premium rates are expensive and allow way more calls than I need. I emailed them to see if they can get me on a lesser plan for cheaper, and will keep this thread updated! Thanks for opening this issue, hopefully I can get this set up soon!
